zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnStateTransitionBehavior.java

```diff
@@ -661,6 +670,12 @@ public <T extends ExecutableFlowElement> void terminateChildProcessInstance(
             () -> containerProcessor.onChildTerminated(element, context, null));
   }
 
+  public static final class ProcessInstanceInterrupted {
+    private static final ProcessInstanceInterrupted INSTANCE = new ProcessInstanceInterrupted();
+
+    private ProcessInstanceInterrupted() {}
+  }
```

Comment:
🔧 This could use some JavaDoc to explain what it is. What does `INSTANCE` mean or how can it be used?